src/main/java/com/google/api/gax/grpc/ApiCallSettingsTyped.java

```diff
@@ -0,0 +1,40 @@
+package com.google.api.gax.grpc;
+
+import io.grpc.MethodDescriptor;
+
+/**
+ * A settings class with generic typing that could used to configrue an API method or create
+ * the method callable object, which can be directly operated against an API.
```

Comment:
"which can be directly operated against an API" -> "which can make API method calls"